src/common/rabbitmq/rabbitmq.module.ts

```diff
@@ -0,0 +1,10 @@
+import { Module } from "@nestjs/common";
+import { ConfigModule } from "@nestjs/config";
+import { RabbitMQService } from "./rabbitmq.service";
+
+@Module({
+  imports: [ConfigModule],
+  providers: [RabbitMQService],
+  exports: [RabbitMQService],
+})
+export class RabbitMQModule {}
```

Comment:
RabbitMQModule is created but is not imported anywhere. Is it intentional?

Reply:
Searching through the source code it seems that CoreJobActionCreators imports [RabbitMQJobActionModule](https://github.com/SciCatProject/scicat-backend-next/blob/954516bc707eb2f941de22118e921b3f3487b93a/src/config/job-config/actions/corejobactioncreators.module.ts)
and [RabbitMQJobActionModule](https://github.com/SciCatProject/scicat-backend-next/blob/954516bc707eb2f941de22118e921b3f3487b93a/src/config/job-config/actions/rabbitmqaction/rabbitmqaction.module.ts) imports CommonModule. 
My guess is, create job action for RabbitMQ works since commonModule imports RabbitMQ service as a provider.